freeswitch/scripts/astpp/scripts/astpp.xml.lua

```diff
@@ -38,6 +38,9 @@ function freeswitch_xml_header(xml,destination_number,accountcode,maxlength,call
 	table.insert(xml, [[<action application="set" data="continue_on_fail=true"/>]]);  
 	--table.insert(xml, [[<action application="set" data="ignore_early_media=true"/>]]);       
 
+	--To check and alter hangup cause
+        table.insert(xml, [[<action application="set" data="execute_on_post_originate=lua alter_hangup_cause.lua ${uuid}"/>]]);
```

Comment:
Hi @devangn 

I found 2 things : 

1. What if someone do not want to change hangup cause? 

2. Change file path. Put that in astpp/lib/.